### DISCUSSION TITLE
name that block

### DISCUSSION THREAD

**Discussion by maceoiscoolunderscorednl** [ScratchAddons/ScratchAddons#7180] [Ideas]
<img width="290" alt="Screenshot 2024-02-18 134725" src="https://github.com/ScratchAddons/ScratchAddons/assets/160402064/7002521c-b142-46f1-9c13-0b1ccc0fc6c6">
imagian being able to name your parts of code so you can find it eazier

**Comment by Samq64** [comment]
Isn't that one of the main things broadcasts and custom blocks (the pink ones) are used for?

**Reply by WorldLanguages** [reply]
Naming through custom blocks should be sufficient most of the time.

**Comment by DNin01** [comment]
Being able to insert "bookmarks" (could be stored internally using comments) might help with navigating code. Many IDEs have features like these - in VS Code, you can use the breadcrumbs to jump to certain parts of your code (which is kind of like what our find bar addon already does) and FUZE4 Nintendo Switch has an implementation of bookmarks.

If this is a new addon, users may not even need the find bar to jump to bookmarks, there could be a bookmarks menu somewhere.
